source/triggers/groups/triggerGroup.js

```diff
@@ -60,8 +60,9 @@ this.checkTriggers$ = function() {
     //  impossible - especially with a low frame rate.  In order to address 
     //  this, do another check after a future(0).
     if ( haveTriggerToFire ) {
-        this.future( 0 ).checkTriggersCallback$();
-    } else if ( this.isChecking ) {
```

Comment:
It looks like isChecking and isEvaluating are two different things.  isChecking enables and disables the trigger group.  If you set it to false, the group will stop checking its triggers entirely, and if you set it to true then it will start again.  isEvaluating tells you whether right now, at this very instant, the trigger group is in the process of doing a check.  Every time a check is done, it first sets isEvaluating to true, then checks all the triggers, then sets isEvaluating back to false.  This prevents the triggers from firing independently of the trigger group (in theory).

I don't think I added isChecking - maybe @BrettASwift did?  In any case, it should have a comment on it - please add one. :)  Also, it seems like the if ( isChecking ) should wrap the entire function (or just force an early exit up at the top), not just prevent you from scheduling the *next* check.  The way it is now, there will always be one more check after you disable it, which doesn't seem desirable.

In any case... this doesn't seem to be a proper fix, though it could indicate that the problem is related to isChecking being enabled/disabled...